### PR TITLE
Assert multiple messages in Messagebus mock backend

### PR DESCRIPTION
**Pull request by simonwahlgren**
## Changes
 - Add new assert function `assert_multiple_messages_produced_with` to help writing cleaner tests to check if multiple messages were produced

### CODE REVIEW

**Comment by gpkc** [comment]
Welcome back

File: tests/messagebus/backends/mock/test_mock_backend.py [hunk 1]
```diff
@@ -31,12 +33,33 @@ def test_consume_messages_should_call_handler(self):
...
+            {'key': 'a', 'value': 'b', 'headers': headers, 'topic': 'foo.bar'},
+        ]
+        self.backend.producer.assert_multiple_messages_produced_with(
+            messages=expected_messages
+        )
+
+        expected_messages.append(
+            {'key': 'b', 'value': 'c', 'headers': headers, 'topic': 'bar.foo'}
+        )
+        with pytest.raises(AssertionError):
```
Comment (gpkc) [inline tests/messagebus/backends/mock/test_mock_backend.py:58]: Should we have a test case for the case where the amount of messages is correct but one of them is wrong?

**Review by gpkc — APPROVED** [review]
LGTM

File: tests/messagebus/backends/mock/test_mock_backend.py (hunk 1 above)
Comment (simonwahlgren) [reply tests/messagebus/backends/mock/test_mock_backend.py:58]: Yes, good idea!

Comment (simonwahlgren) [reply tests/messagebus/backends/mock/test_mock_backend.py:79]: @gpkc Done! Please have a look again

**Review by aericson — APPROVED** [review]

**Pull request merged**